src/index.js

```diff
@@ -45,3 +99,51 @@ $('#cancel-login-button').click(() => {
 $('.logout-button').click(() => {
   window.location = './index.html';
 })
+
+// DATES
+function getTodayDate() {
+  let date = new Date();
+  let month = date.getUTCMonth() + 1;
+  let day = date.getUTCDate();
+  let year = date.getUTCFullYear();
+  if (day < 10 && month < 10) {
+    return `${year}/0${month}/0${day}`;
+  } else if (day < 10 && month >= 10){
+    return `${year}/${month}/0${day}`;
+  } else if (day >= 10 && month < 10){
+    return `${year}/0${month}/${day}`;
+  } else {
+    return `${year}/${month}/${day}`;
+  }
+}
+
+function numifyDate(date) {
+  let year = parseInt(date.split('/')[0]);
+  let month = parseInt(date.split('/')[1]);
+  let day = parseInt(date.split('/')[2]);
+  if (day < 10 && month < 10) {
+    return parseInt(`${year}0${month}0${day}`);
+  } else if (day < 10 && month >= 10){
+    return parseInt(`${year}${month}0${day}`);
+  } else if (day >= 10 && month < 10){
+    return parseInt(`${year}0${month}${day}`);
+  } else {
+    return parseInt(`${year}${month}${day}`);
+  }
+}
+
+function stringifyDate(date) {
```

Comment:
I think you could have used JSON.stringify on numifyDate here, but I love how you break down the logic.